doc/docker.md

```diff
@@ -1,16 +1,102 @@
 # Running with Docker
 
+The advantage of using docker-compose is that it is configured to run several services that the Control Panel relies on, such as the PostgreSQL database and Redis. It initializes these ready for local development purposes.
+
+## Build
+
 To build the docker image, use the following command:
+
 ```sh
 docker build -t controlpanel .
 ```
 
-Running the app requires a PostgreSQL database, and this is provided with
-docker-compose:
+## .env file
+
+You need to create a `.env` file with the settings to enable it to connect to external services.
+
+You probably want it to pick up your 'data' AWS profile:
+```sh
+AWS_PROFILE=data
+```
+
+For testing purposes, you can simply connect to some services used for the dev environment. To do this, for the keys listed below, copy the matching lines from chart-env-config/dev/cpanel.yml and just change the format of each line to `KEY=value`.
+```sh
+# To log in with OIDC
+OIDC_*
+# To interact with AWS account IAM config
+AWS_DATA_ACCOUNT_ID
+AWS_COMPUTE_ACCOUNT_ID
+```
+
+Unless you're testing the slack feature, just disable it with this line:
+```sh
+SLACK_API_TOKEN=disabled
+```
+
+For more details of environment variable settings, refer to: [Environment Variables Reference](environment.md).
+
+## AWS setup
+
+Control Panel needs AWS credentials to change S3 bucket IAM permissions, according to user requests. When this runs in the cluster it can get them using kube2iam. But when you run locally you'll need to provide AWS credentials that give access to the mojanalytics account. Typically you'll use configure to use your Landing Account User and use AssumeRole to the mojanalytics account.
```

Comment:
> Typically you'll use configure to use your Landing Account User and use AssumeRole to the mojanalytics account.

Is "configure" a noun or a `command`..? I'm not sure what that means. Perhaps provide a link to some instructions for me to follow to make this happen.

Reply:
I've reworded this para now